jenkins-pipelines/longevity-ndbench-192-nodes-multidc-4h.jenkinsfile

```diff
@@ -0,0 +1,13 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+longevityPipeline(
+    backend: 'aws',
+    aws_region: '''["eu-west-1", "us-east-1"]''',
+    test_name: 'longevity_test.LongevityTest.test_custom_time',
+    test_config: 'test-cases/longevity/longevity-ndbench-192-nodes-multidc-4h.yaml',
+
+    timeout: [time: 360, unit: 'MINUTES']
```

Comment:
no need to set timeout here, it is set by `vars/getJobTimeouts.groovy`

Reply:
Fixed